straxen/analyses/nveto_plots.py

```diff
@@ -0,0 +1,146 @@
+import straxen
+import strax
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+import os
+
+@straxen.mini_analysis(requires=('raw_records_nv',),
```

Comment:
Hi Daniel, looks nice! Can you add an example of how a pulse should look? Perhaps it's also useful for the MV/TPC? Is it worth writing a wrapper function around this to allow it also to be used for raw_records_x?

Reply:
Ah yes, I forgot to link the example notebook sorry: https://github.com/XENONnT/analysiscode/blob/master/nveto/PulseDisplay.ipynb

> Perhaps it's also useful for the MV/TPC? Is it worth writing a wrapper function around this to allow it also to be used for raw_records_x?
That is actually a good point. We cannot use all features as for the nveto since e.g. the baseline estimated by the digitzer is not know in the other two cases. But let me see if I can make the required changes.

Reply:
Okay addressed your request and updated the MWE notebook.